src/frontend/staff-portal/src/app/components/ticket-status/ticket-status.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { DisputeStatus } from 'app/api';
+
+@Component({
+  selector: 'app-ticket-status',
+  templateUrl: './ticket-status.component.html',
+  styleUrls: ['./ticket-status.component.scss']
+})
+export class TicketStatusComponent implements OnInit {
+  @Input() public dispute: any; // FIXME: this should be of type Dispute, not "any".
+  
+  constructor() { }
+
+  ngOnInit(): void {
+    // FIXME, remove these custom mappings and use the Dispute object directly
+    this.dispute.citizenSubmittedDate = this.dispute.DateSubmitted;
```

Comment:
these two issues are due to the dummy data being passed in from the ticket page.  If you update the dummy data on the dashboard to be a little more production ready then you can remove these custom mappings.

Reply:
Done. Removed these manual mappings to instead just rely on the actual Dispute object properties